cmd/containerd-shim-runhcs-v1/main.go

```diff
@@ -57,6 +57,9 @@ func stack() []byte {
 
 func panicRecover() {
```

Comment:
If we are actually logging panics via the `delete` call now, I don't think we need `panicRecover` anymore. We should be able to just let the shim panic normally and we'll get the log separately, right?

Reply:
Good catch. After I change the `delete` call to log panics via logrus we won't need this.